plugins/catalog/src/apis/EntityPresentationApi/DefaultEntityPresentationApi.ts

```diff
@@ -194,11 +192,12 @@ export class DefaultEntityPresentationApi implements EntityPresentationApi {
     const renderer = options.renderer ?? createDefaultRenderer({ async: true });
 
     const kindIcons: Record<string, IconComponent> = {};
-    Object.entries(options.kindIcons ?? DEFAULT_ICONS).forEach(
-      ([kind, icon]) => {
-        kindIcons[kind.toLocaleLowerCase('en-US')] = icon;
-      },
-    );
+    Object.entries(DEFAULT_ICONS).forEach(([kind, icon]) => {
```

Comment:
not blocking but can't we just merge  these using `{...DEFAULT_ICONS, ...options.kindIcons }` and then do `Object.fromEntries(Object.entries().map(([kind, icon] => [kind.toLocaleLowerCase(), icon]))`?

Reply:
We could for sure! Or even better using mapKeys from lodash